public/app/features/library-panels/types.ts

```diff
@@ -1,17 +1,22 @@
 import { AnyAction } from '@reduxjs/toolkit';
 import { Dispatch } from 'react';
 
+import { LibraryPanel } from '@grafana/schema';
+import { LibraryElementDTOMetaUser } from '@grafana/schema/src/raw/librarypanel/x/librarypanel_types.gen';
+
 import { PanelModel } from '../dashboard/state';
 
 export enum LibraryElementKind {
   Panel = 1,
-  Variable,
 }
 
 export enum LibraryElementConnectionKind {
   Dashboard = 1,
 }
 
+/** @deprecated use LibraryPanel */
+export interface LibraryElementDTO extends LibraryPanel {}
+
```

Comment:
Since it's a core type, maybe we can just remove it and update usage to use `LibraryPanel` directly?

Reply:
Yes -- likely a good plan, but that started touching almost everything so I punted for now :)